src/common/filesystembase.h

```diff
@@ -107,8 +107,11 @@ namespace FileSystem {
      * Equivalent to QFile::remove(), except on Windows, where it will also
      * successfully remove read-only files.
      */
-    bool OCSYNC_EXPORT remove(const QString &fileName, QString *errorString = 0);
+    bool OCSYNC_EXPORT remove(const QString &fileName, QString *errorString = 0, bool _moveToTrash = false);
```

Comment:
coding style:  '_' at the beginning is indicating a member. So there should not be an _ there.